client/static/css/style.css

```diff
@@ -0,0 +1,64 @@
+section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+#logo {
+    margin-top: 100px;
+    margin-bottom: 0px;
+}
+
+#searchBar {
+    margin: 50px 0px 20px 0px;
+    padding: 10px 15px 10px 15px;
+    width: 600px;
+    background-color: transparent;
+    
+    border-radius: 20px;
+    border-width: 0.5px;
+    border-color: #D3D3D3;
+
+    font-family: 'Source Sans Pro', sans-serif;
+    font-size: 18px;
+}
+
+#search, #surprise {
+    font-family: 'Source Sans Pro', sans-serif;
+    font-size: 15px;
+
+    padding: 10px 25px 10px 25px;
+    margin: 10px 15px 10px 15px;
+    border: none;
+    border-radius: 3px;
+    width: 125px;
+}
+
+h1 {
+    font-size: 25px;
+    font-family: 'Source Sans Pro', sans-serif;
+}
+
+a {
+    margin-bottom: 5px;
+
+    font-size: 25px;
+    font-family: 'Source Sans Pro', sans-serif;
+    color: #273BE2
+}
+
+p {
+    margin-top: 2px;
+    font-family: 'Source Sans Pro', sans-serif;
+}
+
+.hide {
+   position: absolute;
+   left: -999px;
+}
+
+.moveUp {
+    position: sticky;
+}
+
```

Comment:
If you have time, it can be made more responsive using rem measurements, rather than px.